elfi/methods/parameter_inference.py

```diff
@@ -7,6 +7,9 @@
 
```

Comment:
It's fine to keep the CopulaABC implementation in its own file. The old `parameter_inference.py` is getting a bit too long anyway (for my taste at least).